test/bootc-sources/microshift_repo_config.sh.template

```diff
@@ -71,18 +71,25 @@ sslclientcert = ${sslcrt}
 skip_if_unavailable=0
 EOF
 
-    # Only the y-2 version is guaranteed to be available for released repositories
     cat > "${OCP_RHOCP_REPO_FILE}" <<EOF
 [openshift-rhocp]
 name=OpenShift Dependencies RHEL 9
-baseurl=https://cdn.redhat.com/content/dist/layered/rhel9/{{ .Env.UNAME_M }}/rhocp/4.{{ .Env.YMINUS2_MINOR_VERSION }}/os
+{{- if env.Getenv "RHOCP_MINOR_Y" "" }}
+baseurl=https://cdn.redhat.com/content/dist/layered/rhel9/{{ .Env.UNAME_M }}/rhocp/4.{{ .Env.RHOCP_MINOR_Y }}/os
 enabled=1
 gpgcheck=1
 gpgkey = file:///etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
 sslverify=1
 sslcacert = /etc/rhsm/ca/redhat-uep.pem
 sslclientkey = ${sslkey}
 sslclientcert = ${sslcrt}
+{{- end }}
+{{- if env.Getenv "RHOCP_MINOR_Y_BETA" "" }}
```

Comment:
Is there `else if`?
Generally, we should generate an error if neither of these is defined.

Reply:
Oh, yes, I missed `else if` in the docs...
Hmm, we can generate error if we try using RHOCP_MINOR_Y_BETA without check if it exists (straight in the `else`), what do you think?